libraries/ms-common/src/x86/adc.c

```diff
@@ -1,23 +1,172 @@
+#include <stddef.h>
+
 #include "adc.h"
+#include "interrupt.h"
+#include "soft_timer.h"
+// x86 implementation very similar to STM32F0 implementation.
+// adc_read_raw should always return 4090.
+// Vdda locked at 3300 mV.
+// adc_read_converted should always return close to 2V
+
+#define ADC_RETURNED_VOLTAGE_RAW 2500
+#define ADC_CONTINUOUS_CB_FREQ_MS 50
+
+// TS_CAL addresses obtained from section 3.10.1 of the specific device
+// datasheet
+#define ADC_TS_CAL1 0x1FFFF7b8
+#define ADC_TS_CAL2 0x1FFFF7c2
```

Comment:
Like I commented below, these hardcoded pointers won't work on x86. Should be removed, I'd replaced them with hardcoded `TS_CAL` values.